dev/docs/windows-source-installation.md

```diff
@@ -0,0 +1,81 @@
+# Developing Apache SystemDS on Windows Platform
+
+These instructions will help you build Apache SystemDS from source code, which is the basis for engine and algorithms development.
+The following conventions will be used to refer to directories on your machine:
+
+* `<USER_HOME>` is your home directory.
+* `<MAVEN_HOME>` is the root directory for the Apache Maven source code.  
+* `<SYSTEMDS_HOME>` is the root directory for the SystemDS source code.
+* `<SPARK_HOME>` is the root directory for the Apache Spark source code.
+* `<SCALA_HOME>` is the root directory for the Scala.
+Ex. `C:\Program Files (x86)\scala`
+* `<SBT_HOME>` is the root directory for the Scala SBT.
+Ex. `C:\Program Files (x86\sbt`
+* `<JDK_18_HOME>` is the root directory for the 1.8 JDK.
+* `<CUDA_HOME>`,`<CUDA_PATH>` is the top directory for NVIDIA GPU Computing Toolkit.  
+Ex. `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0`
+Also, make sure that `libnvvp`, `bin` folders are accessible to command line.
+*  `<CUDNN_PATH>` is the path 
+Ex. `C:\Program Files\cudnn-9.0-windows10-x64-v7.2.1.38\cuda`
```

Comment:
I think this cudnn version is depricated, and we use 10.1 now.
I think a better suggestion is to have a linked folder (if that exists in windows) thereby removing the knowledge about version in the guide

Reply:
Is this about `mklink` to create symbolic links. Will try to do. :)

Let us ask about version in the dev list.

Reply:
I don't think that we need to discuss it on dev list, since it is already updated on master, and this is just out of date.